draft-ietf-v6ops-prefer8781.md

```diff
@@ -46,103 +46,108 @@ informative:
   RFC4861:
   RFC4862:
   RFC6105:
+  RFC6052:
   RFC6144:
   RFC6146:
   RFC7915:
   RFC6147:
   RFC6877:
+  RFC7051:
   RFC8880:
   RFC9463:
   RFC9499:
   RFC8305:
 
 --- abstract
 
-On networks providing IPv4-IPv6 translation (NAT64, RFC7915), hosts and other endpoints might need to know the IPv6 prefix used for translation (the NAT64 prefix).
-While RFC7050 defined a DNS64-based prefix discovery mechanism, more robust methods have since emerged.
-This document provides updated guidelines for NAT64 prefix discovery, deprecating the RFC7050 approach in favor of modern alternatives (e.g., RFC8781) whenever available.
+On networks providing IPv4-IPv6 translation (RFC7915), hosts and other endpoints might need to know the IPv6 prefix(es) used for translation (the NAT64 prefix).
+While "Discovery of the IPv6 Prefix Used for IPv6 Address Synthesis" (RFC7050) defines a DNS64-based prefix discovery mechanism, more robust methods have been developed since then.
+This document provides guidelines for NAT64 prefix discovery, recommending more deterministic alternatives like RFC8781 over RFC7050 when available.
```

Comment:
Since this is the first mention of 8781, I would add the title in front of it like you did with 7050 above (a great practice).